src/test/java/nl/tudelft/ewi/devhub/webtests/views/AssignmentView.java

```diff
@@ -0,0 +1,126 @@
+package nl.tudelft.ewi.devhub.webtests.views;
+
+import lombok.AccessLevel;
+import lombok.Data;
+import lombok.Getter;
+import nl.tudelft.ewi.devhub.server.database.entities.Delivery;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.List;
+import java.util.Locale;
+
+import javax.annotation.Nullable;
+
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+/**
+ * Created by Douwe Koopmans on 4-5-16.
+ */
+public class AssignmentView extends ProjectSidebarView {
+
+    private static final By RECENT_ASSIGNMENTS_HEADER = By.xpath("//h4[starts-with(normalize-space(.), 'Assignments')]");
+    public AssignmentView(WebDriver driver) {
+        super(driver);
+    }
+    
+    @Override
+    protected void invariant() {
+        super.invariant();
+        assertNotNull(getDriver().findElement(RECENT_ASSIGNMENTS_HEADER));
+        assertTrue(currentPathStartsWith("/courses/ti1705/TI1705/groups/"));
+    }
+
+    public Assignment getAssignment() throws ParseException {
+        return new Assignment(getDriver().findElement(By.cssSelector("a[href^=\"deliveries\"]")));
+    }
+
+    @Data
+    public class Assignment {
+        @Getter(AccessLevel.NONE)
+        private final WebElement anchor;
+        private final Review review = new Review();
+
+        public String getAuthor() {
+            return getDriver().findElements(By.tagName("dd")).get(0).getText();
+        }
+
+        public String getName() {
+            return getDriver().findElement(By.cssSelector("div .col-md-offset-2.col-md-10 h4")).getText();
+        }
+
+        public Delivery.State getStatus() {
+            return Delivery.State.valueOf(getDriver().findElement(By.cssSelector("span.label")).getText().toUpperCase());
+        }
+
+        public Date getDueDate() throws ParseException {
+            return new SimpleDateFormat("EEEE dd MMMM yyyy k:m", Locale.US).parse(
+                    getDriver().findElements(By.tagName("dd")).get(1).getText());
+        }
+
+        public DeliveryReviewView click() {
```

Comment:
Its good to create an `Assignment` class that is able to resolve a few properties of the assignment. However, navigating to the `DeliveryReviewView` is specific for the assignment submission/delivery, and not for the assignment. Furthermore, the method name `click` is not very descriptive for going to the review view here (in contract to `getAssignments().get().click()` where you at least have some idea that you will navigate to the assignments page).